Goobi/src/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -1288,13 +1290,29 @@ public void setBatchHelper(BatchStepHelper batchHelper) {
     public List<ProcessProperty> getContainerlessProperties() {
         List<ProcessProperty> answer = new ArrayList<ProcessProperty>();
         for (ProcessProperty pp : this.processPropertyList) {
-            if (pp.getContainer() == 0) {
+            if (pp.getContainer() == 0 && (isEmptyString(pp.getValue())
+                    || !getOccupiedProperties().contains(pp.getName()))) {
                 answer.add(pp);
             }
         }
         return answer;
     }
 
+    private Set<String> getOccupiedProperties() {
+        Set<String> propertiesMitWerten = new HashSet<>();
+        for (ProcessProperty pp : processPropertyList)
```

Comment:
Please use bracket here in in the next lines.

Reply:
Blocks were added.